src/Domain/CoordinateMaps/CoordinateMap.tpp

```diff
@@ -557,8 +541,9 @@ auto CoordinateMap<SourceFrame, TargetFrame, Maps...>::
                     InverseJacobian<T, dim, SourceFrame, TargetFrame>,
                     Jacobian<T, dim, SourceFrame, TargetFrame>,
                     tnsr::I<T, dim, TargetFrame>>{
-      tnsr::I<T, dim, TargetFrame>(std::move(mapped_point)), std::move(inv_jac),
-      std::move(jac), std::move(frame_velocity)};
+      tnsr::I<T, dim, TargetFrame>(std::move(mapped_point)),
+      determinant_and_inverse(jac).second, std::move(jac),
```

Comment:
Make sure to add a comment that this done numerically on purpose and was profiled.